pkg/api/frontendsettings.go

```diff
@@ -63,6 +64,10 @@ func (hs *HTTPServer) getFrontendSettings(c *contextmodel.ReqContext) (*dtos.Fro
 			continue
 		}
 
+		if panel.ID == "datagrid" && !hs.Features.IsEnabled(featuremgmt.FlagEnableDatagridEditing) {
```

Comment:
Nitpick: I like to be explicit with my booleans just to make it super clear to anyone reading it, and to the IsEnabled function; I'd !! it. -> `!hs.Features.IsEnabled(!!featuremgmt.FlagEnableDatagridEditing)`

Reply:
Where are you seeing that the `FlagEnableDatagridEditing` can be undefined? Go doesn't even have undefined.